pkg/registry/core/rest/storage_core.go

```diff
@@ -179,50 +175,90 @@ func (c LegacyRESTStorageProvider) NewLegacyRESTStorage(restOptionsGetter generi
 
 	serviceRest := service.NewStorage(serviceRegistry, endpointRegistry, ServiceClusterIPAllocator, ServiceNodePortAllocator, c.ProxyTransport)
 
-	restStorageMap := map[string]rest.Storage{
```

Comment:
This change is really significant.  Disabling bits of the core API isn't a thing that was generally support before.  I remember the code only allowed it for extensions and even that was probably a mistake.

I don't think that you should be able to pick and choose from groups as a rule.